docs/quick-lint-js.schema.config

```diff
@@ -0,0 +1,65 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
This link doesn't load in Firefox for me. Is this the right link?

Reply:
It is working for me in Firefox and Chrome. It downloads a schema file.